[IMP] ✨ Adapted the energy_communities_cooperator module to the new changes in the cooperator module.

AlvaroGRuiz opened the pull request:
[IMP] ✨ Adapted the energy_communities_cooperator module to the new changes in the cooperator module.
OP#1495 
…

Author: Alvaro Garcia <alvaro.garcia@somit.coop>

AlvaroGRuiz committed:
[IMP] ✨ Adapted the energy_communities_cooperator module to the new changes in the cooperator module.

Author: Alvaro Garcia <alvaro.garcia@somit.coop>

daniquilez commented on energy_communities_cooperator/__manifest__.py:
```diff
@@ -48,6 +48,7 @@
         "views/voluntary_share_interest_return_views.xml",
         "views/subscription_request_view.xml",
         "views/website_subscription_template.xml",
+        # "views/res_config_settings_view.xml",
```
why is this view still not included? I see the view definition below on the PR?

juan-pe replied:
done!

AlvaroGRuiz committed:
[FIX] 🐛 All new fields of res_config_setting  must be related to res_company fields

Author: Alvaro Garcia <alvaro.garcia@somit.coop>

AlvaroGRuiz committed:
[IMP] ✨ Add all new fields of cooperator on model res_company to res_config_settings

Author: Alvaro Garcia <alvaro.garcia@somit.coop>
Author: Dani Quilez <dani.quilez@somenergia.coop>
Author: Juan Pedro Sánchez <juanpe.sanchez@somenergia.coop>

AlvaroGRuiz committed:
[REL] 🏗️  New release for energy_communities_cooperator to 16.0.0.5.0

Author: Alvaro Garcia <alvaro.garcia@somit.coop>
Author: Juan Pedro Sánchez <juanpe.sanchez@somenergia.coop>

juan-pe reviewed (approved):
🎉
